File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if(nums.size() == 0){
            return 0;
        }
        
        int curr = 0;
        for(int i=1; i<nums.size(); i++){
            if(nums[i] != nums[curr]){
                curr++;
            nums[curr] = nums[i];
            }
        }
        
        // Returning Number Of Unique Values
        return curr+1;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
            
        int i=0 , j=0;
        
        while(j < nums.length) {
            while(j < nums.length && nums[i] == nums[j]) {
                j++;
            }
            
            if(j < nums.length) {
                nums[i+1] = nums[j];
                i++;
            }
        }
        
        return i+1;
    }
}